src/components/Home/TransactionCard/TransactionCard.tsx

```diff
@@ -1,43 +1,47 @@
 import { FC, useContext, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { ReactComponent as ArrowDownIcon } from "../../../assets/arrow-down.svg";
+import { ReactComponent as InfoCircleIcon } from "../../../assets/info-circle.svg";
 import { ReactComponent as ClosedLockIcon } from "../../../assets/lock-closed.svg";
 import { Transaction } from "../../../models/transaction.model";
 import { AppContext } from "../../../store/app-context";
 import LoadingBox from "../../Shared/LoadingBox/LoadingBox";
 import SingleTransaction from "./SingleTransaction/SingleTransaction";
 
-type Props = {
+export type Props = {
```

Comment:
is the export necessary?

Reply:
Using these in the test: https://github.com/cstenglein/raspiblitz-web/pull/175/files#diff-0f0ef366c5b649162bb1970530376b1278e07f4bcdcc415a07c36e34eaddbf1cR4